cumulusci/cli/cci.py

```diff
@@ -1009,6 +1012,27 @@ def org_scratch_delete(config, org_name):
     config.keychain.set_org(org_config)
 
 
+@click.command(
+    name="shell",
+    help="Drop into a Python shell with a simple_salesforce connection in `sf`",
+)
+@click.argument("org_name", required=False)
+@pass_config
+def org_shell(config, org_name):
+    org_name, org_config = config.get_org(org_name)
+    org_config.refresh_oauth_token(config.keychain)
+
+    sf = get_simple_salesforce_connection(config.project_config, org_config)
+
+    code.interact(
+        banner="Use `sf` to access org `{}` via simple_salesforce".format(org_name),
+        local=dict(globals(), **locals()),
```

Comment:
Why include the `cci` module's globals here?

Should we include a direct reference to `project_config` too?

Reply:
I assumed (without a good reason!) that there was a design decision behind doing so in the `shell` command that would also apply here - but if that command does so more with an eye towards facilitating CCI debugging, there's no need to do so here, and exposing just `sf` and `project_config` (perhaps also a `Salesforce_Bulk` instance) would be enough.

Reply:
I'd suggest exposing org_config too.

Reply:
Done!